bundles/binding/org.openhab.binding.digitalstrom/src/main/java/org/openhab/binding/digitalstrom/internal/DigitalSTROMGenericBindingProvider.java

```diff
@@ -121,9 +119,4 @@ public void removeConfigurations(String context) {
 		}
 		return deviceConsumptionItems;
 	}
-
-	@Override
-	public Set<Item> getItemNamesByContext(String context) {
-		return new HashMap<String, Set<Item>>(super.contextMap).get(context);
-	}
-}
```

Comment:
Removed unused `getItemNamesByContext` method since it exposes non-threadsafe Set of Items.